src/applications/login/containers/SignInApp.jsx

```diff
@@ -0,0 +1,233 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import ExternalServicesError from 'platform/monitoring/external-services/ExternalServicesError';
+// import recordEvent from 'platform/monitoring/record-event';
+import SubmitSignInForm from 'platform/static-data/SubmitSignInForm';
+import { login, signup } from 'platform/user/authentication/utilities';
+import environment from 'platform/utilities/environment';
+
+import downtimeBanners from '../utilities/downtimeBanners';
+
+const loginHandler = loginType => () => {
+  // TODO add separate login tracking for this page
+  //   recordEvent({ event: `login-attempted-${loginType}` });
+  login(loginType, 'v1');
+};
+
+const handleDsLogon = loginHandler('dslogon');
+const handleMhv = loginHandler('mhv');
+const handleIdMe = loginHandler('idme');
+
+const vaGovFullDomain = environment.BASE_URL;
+
+class SignInModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      globalDowntime: false,
+    };
+
+    this.setGlobalDowntimeState = this.setGlobalDowntimeState.bind(this);
+  }
+
+  setGlobalDowntimeState() {
+    this.setState({ globalDowntime: true });
+  }
```

Comment:
You can use `setGlobalDowntimeState = () => {` and then get rid of the `bind` on line 33.

Reply:
nice, I was thinking that I would modify that, but I recall it requiring a babel transform in the past, and didn't know if it still did and if we had it in our build setup